packages/gatsby-theme-docs/gatsby-browser.js

```diff
@@ -6,3 +6,15 @@
 
 require('prism-themes/themes/prism-dracula.css');
 require('prismjs/plugins/command-line/prism-command-line.css');
+
+exports.onClientEntry = () => {
+  require.ensure(['@sentry/browser'], require => {
+    const Sentry = require('@sentry/browser');
+    Sentry.init({
+      dsn: 'https://e43538aae75e412eb16b27d8011f5a8b@sentry.io/1819068',
```

Comment:
I set up the sentry project "docs" in our commercetools account. The idea is that all websites use the same setting.